### DISCUSSION TITLE
How to implement password recovery in Flutter using Supabase?

### DISCUSSION THREAD

**Discussion by AIQMEDIA** [supabase/supabase#2242] [Questions]
Hi, I am currently building an app that has a forgot password option which when clicked should allow the user to change the password.
I am using below API provided by Supabase :
await auth.api.resetPasswordForEmail('example@email.com')
However, this redirects to a page with below error : 

localhost refused to connect

I am made to understand that I have to update SITE_URL so as the above api redirects back to my app and then the app is able to detect token=recovery and send another instruction for password to be updated in supabase database.

But this is my first time working on supabase and I don't know what exactly to put in the SITE_URL and how to make it redirect to my app and implement the password recovery successfully in my app. If anyone has already implemented an example in flutter, could you please link it down below ?

**Comment by kiwicopple** [comment]
Hi @AIQMEDIA - you can put the SITE_URL in your Auth settings here:

![image](https://user-images.githubusercontent.com/10214025/124760864-bff20e00-df63-11eb-86fa-f2a6e11fd316.png)

> how to make it redirect to my app and implement the password recovery successfully in my app

You'll need to implement deep linking! https://stackoverflow.com/questions/53770563/how-to-open-flutter-application-from-url
